cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java

```diff
@@ -127,6 +125,34 @@
    */
   private int readDataConsistencyLevel = 1;
 
+  /**
+   * Max num of threads can concurrently execute qp sub-tasks of all client requests. When <= 0, use
+   * CPU core number * 10. Each client request corresponds to a QP Task. A QP task may be divided
+   * into several sub-tasks.
+   */
+  private int concurrentQPSubTaskThread = Runtime.getRuntime().availableProcessors() * 10;
```

Comment:
Maximum number of threads which execute tasks generated by client requests concurrently.
Each client request corresponds to a QP Task. A QP task may be divided into several sub-tasks. 
So this value is the sum of all sub-tasks.
When this value <= 0, use CPU core number * 10.

Reply:
Accept